luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/LuthierIndustrialPark.java

```diff
@@ -409,8 +423,73 @@ public Builder(LuthierResourceDictionaries resourceDictionaries, GranularityDict
         }
 
-        public GranularityDictionary getGranularityDictionary() {
-            return granularityDictionary;
+        /**
+         * Default LogicalTable factories that are defined in fili-core.
+         *
+         * @return  a Map of MetricMaker name to its factory
+         */
+        private Map<String, Factory<LogicalMetric>> getDefaultMetricFactories() {
+            Map<String, Factory<LogicalMetric>> metricFactoryMap = new LinkedHashMap<>();
+            DefaultMetricFactory defaultMetricFactory = new DefaultMetricFactory();
+            /* short aliases */
+            metricFactoryMap.put("default", defaultMetricFactory);
+            /* class names */
+            metricFactoryMap.put("LogicalMetric", defaultMetricFactory);
+            /* fully qualified class names */
+
+            metricFactoryMap.put("com.yahoo.bard.webservice.data.metric.LogicalMetric", defaultMetricFactory);
+            return metricFactoryMap;
+        }
+
+        /**
+         * Default LogicalTable factories that are defined in fili-core.
+         *
+         * @return  a Map of MetricMaker name to its factory
+         */
+        private Map<String, Factory<MetricMaker>> getDefaultMetricMakerFactories() {
+            Map<String, Factory<MetricMaker>> metricMakerFactoryMap = new LinkedHashMap<>();
+            ArithmeticMakerFactory arithmeticMakerFactory = new ArithmeticMakerFactory();
+            LongSumMakerFactory longSumMakerFactory = new LongSumMakerFactory();
+            DoubleSumMakerFactory doubleSumMakerFactory = new DoubleSumMakerFactory();
+            AggregationAverageMakerFactory aggregationAvgMakerFactory = new AggregationAverageMakerFactory();
+            CountMakerFactory countMakerFactory = new CountMakerFactory();
+            /* short aliases */
+            metricMakerFactoryMap.put("arithmetic", arithmeticMakerFactory);
+            metricMakerFactoryMap.put("longSum", longSumMakerFactory);
+            metricMakerFactoryMap.put("doubleSum", doubleSumMakerFactory);
+            metricMakerFactoryMap.put("avg", aggregationAvgMakerFactory);
+            metricMakerFactoryMap.put("average", aggregationAvgMakerFactory);
+            metricMakerFactoryMap.put("aggregateAverage", aggregationAvgMakerFactory);
+            metricMakerFactoryMap.put("count", countMakerFactory);
+            /* class names */
+            metricMakerFactoryMap.put("ArithmeticMaker", arithmeticMakerFactory);
+            metricMakerFactoryMap.put("LongSumMaker", longSumMakerFactory);
+            metricMakerFactoryMap.put("DoubleSumMaker", doubleSumMakerFactory);
+            metricMakerFactoryMap.put("AggregationAverageMaker", aggregationAvgMakerFactory);
+            metricMakerFactoryMap.put("CountMakerFactory", countMakerFactory);
+            /* fully qualified class names */
+            metricMakerFactoryMap.put(
+                    "com.yahoo.bard.webservice.data.config.metric.makers.ArithmeticMaker",
```

Comment:
Rather than hardcoding all of these, we should use things like `ArithmeticMaker.class.getName()`. Less prone to typos.

Reply:
Used a much more improved (?) function call to replace a lot of mindless work, while incorporating your idea.